server/fleet/errors.go

```diff
@@ -546,6 +546,7 @@ const (
 	RunScriptScriptsDisabledGloballyErrMsg = "Running scripts is disabled in organization settings."
 	RunScriptDisabledErrMsg                = "Scripts are disabled for this host. To run scripts, deploy the fleetd agent with scripts enabled."
 	RunScriptScriptTimeoutErrMsg           = "Timeout. Fleet stopped the script after 5 minutes to protect host performance."
+	RunScriptAsyncScriptEnqueuedErrMsg     = "Script is running or will run when the host comes online."
```

Comment:
I added this string here because it seemed similar to all the others, but it's also not really an
"error" per se. Happy to hear if there's a better place for this!